### PR TITLE
Dvfs chown

### PR DESCRIPTION
**Pull request by anton-bondarev**
Implement chown() syscall and make chown command posix compatible

### CODE REVIEW

File: src/compat/posix/fs/chown/chown_dvfs.c [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	}
+
+	res = dvfs_lookup(path, &lookup);
+	if (res) {
+		return SET_ERRNO(ENOENT);
+	}
+	assert(lookup.item);
+	assert(lookup.item->d_inode);
+	lookup.item->d_inode->i_owner_id = owner;
+	lookup.item->d_inode->i_owner_id = group;
```
Comment (alexkalmuk) [inline src/compat/posix/fs/chown/chown_dvfs.c]: group_id

**Comment by alexkalmuk** [comment]
LGTM

File: src/cmds/fs/chown/chown.c [hunk 2]
```diff
@@ -5,30 +5,35 @@
...
+		struct stat st_buf;
 
-		ret = vfs_lookup(file, &node_path);
-		if (ret) {
-			fprintf(stderr, "Can't open %s: %s\n", file, strerror(-ret));
-			return ret;
+		if(!is_group_set) {
+			res = stat(file, &st_buf);
+			if (res) {
+				fprintf(stderr, "stat(%s, ..) return error = %d\n",
```
Comment (Deryugin) [inline src/cmds/fs/chown/chown.c]: Should we use logger here?

File: src/compat/posix/fs/chown/chown_dvfs.c (hunk 1 above)
Comment (Deryugin) [inline src/compat/posix/fs/chown/chown_dvfs.c]: Is actual inode information being updated on the disk?

**Comment by Deryugin** [comment]
LGTM

**Pull request merged**